externs/ngeox.js

```diff
@@ -258,6 +258,43 @@ ngeox.format.FeatureHashOptions.prototype.properties;
  */
 ngeox.interaction;
 
+
+/**
+ * @typedef {{
+ *     pixelTolerance: (number|undefined),
+ *     style: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *     features: ol.Collection.<ol.Feature>,
+ *     wrapX: (boolean|undefined)}}
+ * @api
+ */
+ngeox.interaction.ModifyOptions;
```

Comment:
Oh right, I'm trying it now, I think you're right. I had done it in another commit after the export one because I still had an error, but that must have been the missing ngettext error.